src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -371,7 +371,7 @@ void func_80AEB1D8(EnRu1* this) {
     this->actor.speed = 0.0f;
     this->actor.gravity = 0.0f;
     this->actor.minVelocityY = 0.0f;
-    func_80AEB0EC(this, 0);
+    EnRu1_SetCameraSetting(this, 0);
```

Comment:
I would suggest to make an enum, but ultimately i think this will be an enum provided by the scene file, whenever we document those someday. Maybe at a TODO comment at one of the functions?